.travis.yml

```diff
@@ -7,10 +7,10 @@ install:
  - git clone https://github.com/Hackerpilot/Dscanner
  - (cd Dscanner && git checkout tags/v0.4.0-alpha2)
  - (cd Dscanner && git submodule update --init --recursive)
- - (cd Dscanner && make githash debug)
+ - (cd Dscanner && sed 's/dparse_verbose/StdLoggerDisableWarning/' -i makefile && make githash debug)
  - mv Dscanner/dsc .
  - rm -rf Dscanner
 script:
  - (cd .. && rdmd ./checkwhitespace.d $(find phobos -name '*.d'))
  - grep -E "(for|foreach|foreach_reverse|if|while)\(" $(find . -name '*.d'); test $? -eq 1
- - dscanResult=$(./dscanner --config .dscanner.ini --styleCheck std etc 2> /dev/null | grep -vE "(Empty declaration|Primary expression expected)") ; if [ -z "${dscanResult}" ] ; then echo "No warnings found" ; else echo $dscanResult && $(exit 1); fi
+ - ./dsc --config .dscanner.ini --styleCheck $(find std -type f -name '*.d' | grep -vE 'std/c/process.d|std/outbuffer.d|std/stream.d|std/traits.d|std/typecons.d|std/conv.d')
```

Comment:
Also it appears that there's a bug causing D-Scanner to not exit with a value of 1 when problems are found, so this won't fail correctly.